config/environments/development.rb

```diff
@@ -78,7 +78,7 @@
     enable_starttls_auto: true
   }
 
-  # config.action_mailer.delivery_method = :letter_opener
+  config.action_mailer.delivery_method = :letter_opener
   config.action_mailer.perform_deliveries = true
-  HOST_URL = 'http://localhost:8080'
```

Comment:
Do not change this, even production.rb has a different definition for HOST_URL.

Reply:
https://github.com/joshsoftware/providesk_api/commit/62391a5494548a9fe8de209564c78b0dd2ea8990
referred this pr that was previously there this host url is defined for mail only so i just changed the localhost to 3000 for production the url is same as previous.